calico-cloud/multicluster/kubeconfig.mdx

```diff
@@ -340,6 +340,14 @@ spec:
 
 Congratulations! You have completed configuration of federated endpoint identity.
 
+:::note
+
+Openshift users are advised to disable the `network.openshift.io/RestrictedEndpointsAdmission` admission controller
```

Comment:
I think you should be more forceful.  IIUC, service IP federation simply doesn't work with that controller enabled so we should be explicit.